interpreter/CppInterOp/unittests/CppInterOp/ScopeReflectionTest.cpp

```diff
@@ -158,7 +158,7 @@ TEST(ScopeReflectionTest, SizeOf) {
   EXPECT_EQ(Cpp::SizeOf(Decls[4]), (size_t)0);
   EXPECT_EQ(Cpp::SizeOf(Decls[5]), (size_t)1);
   EXPECT_EQ(Cpp::SizeOf(Decls[6]), (size_t)4);
-  EXPECT_EQ(Cpp::SizeOf(Decls[7]), (size_t)16);
+  EXPECT_EQ(Cpp::SizeOf(Decls[7]), (size_t) (sizeof(size_t) == 4 ? 12 : 16));
```

Comment:
Based on the CI failure, I believe we also need to account for 32-bit MSVC platforms since the double is 8-byte aligned.

Perhaps an approach like: 
```suggestion
  struct B {short a; double b;};
  EXPECT_EQ(Cpp::SizeOf(Decls[7]), sizeof(B));
```
should always reflect the correct compiler ABI